src/components/userDasboard/container/UserDashboard.jsx

```diff
@@ -47,7 +47,7 @@ export class UserDashboard extends Component {
       orgDictionary,
       loading,
     } = this.props;
-    const dictionary = public_collections < 2 ? 'dictionary' : 'dictionaries';
```

Comment:
What do you intend to achieve with this change?

Reply:
Well dictionary will only be shown once when there is only one dictionary

Reply:
ok, cool.